### ISSUE TITLE
Disable re-create like ViewPager

### ISSUE THREAD

**Issue by toledorobson** [SimformSolutionsPvtLtd/SSCustomBottomNavigation#30]
Hi, thanks for this library. 

I added my fragments with jetpack navigation but when i change tab from the menu, fragments are re-created. I want to continue where I left off when i reselect a menu item like viewpager.

Is that possible? Thank you.

**Comment by MehulKSimform** [comment]
@toledorobson  : could you please share more info.

**Comment by MehulKK** [comment]
@toledorobson  : would you please share more details for this.

**Comment by sagarishtech** [comment]
Yes i have same issue, how to prevent Fragment refreshing Every time when we clicked on menu item. So kindly tell us how to prevent Refreshing fragments every time?

ex: We have 4 fragments A, B, C, D => currently we are on Fragment A, so when we clicked on fragment B and again i click back on Fragment A, here i dont want to Recreate fragment A, Same as all others fragments.

is this possible?

Thank you
